# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
#
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
#
# ROCK Linux: rock-src/package/x11/xfree86/macintosh-xkb-us-de-tweak.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
#
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
#
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
#
# --- ROCK-COPYRIGHT-NOTE-END ---

Otherwise all the third and fourth level symbols are not reachable on
iBooks :-(((

  Rene Rebe <rene@rocklinux.org>

--- xc/programs/xkbcomp/symbols/macintosh/de	2003-06-06 02:32:26.000000000 +0200
+++ xc/programs/xkbcomp/symbols/macintosh/de	2003-06-08 13:15:36.000000000 +0200
@@ -89,11 +89,13 @@
     key <KPDL> {	[  KP_Delete,	KP_Separator	]	};
     // End "Keypad" section
 
+    key <LALT> {        [ Mode_switch,  Multi_key       ]       };
+
     // begin modifier mappings
 //    modifier_map Shift  { Shift_L };
 //    modifier_map Lock   { Caps_Lock };
 //    modifier_map Control{ Control_L };
-//    modifier_map Mod3	{ Mode_switch };
+    modifier_map Mod3	{ Mode_switch };
 };
 
 partial alphanumeric_keys 
--- xc/programs/xkbcomp/symbols/macintosh/us	2003-06-06 02:32:26.000000000 +0200
+++ xc/programs/xkbcomp/symbols/macintosh/us	2003-06-08 13:19:58.000000000 +0200
@@ -22,7 +22,7 @@
     key <AE10> {	[	  0,	parenright	]	};
     key <AE11> {	[     minus,	underscore	]	};
     key <AE12> {	[     equal,	plus		]	};
-    key <BKSP> {	[ BackSpace			]	};
+    key <BKSP> {	[ BackSpace,    KP_Delete	]	};
 
     key  <TAB> {	[	Tab,	ISO_Left_Tab	]	};
     key <AD01> {	[	  q,	Q 		]	};
@@ -129,10 +129,10 @@
     key <RTSH> {	[  Shift_R	]	};
     key <LCTL> {	[  Control_L	]	};
     key <RCTL> {	[  Control_R	]	};
-    key <LALT> {	[  Alt_L	]	};
-    key <RALT> {	[  Alt_R	]	};
-    key <LWIN> {	[  Meta_L	]	};
-    key <WWIN> {	[  Meta_R	]	};
+    key <LWIN> {	[  Alt_L,  Meta_L	]	};
+    key <RWIN> {	[  Alt_R,  Meta_R	]	};
+#    key <LWIN> {	[  Meta_L	]	};
+#    key <WWIN> {	[  Meta_R	]	};
     // End "Modifier" section
 
     // begin modifier mappings
@@ -141,7 +141,7 @@
     modifier_map Control{ Control_L, Control_R };
     modifier_map Mod1   { Alt_L, Alt_R };
     modifier_map Mod2   { Num_Lock };
-    modifier_map Mod4   { Meta_L, Meta_R };
+#    modifier_map Mod4   { Meta_L, Meta_R };
     modifier_map Mod5   { Scroll_Lock };
     // end modifier mappings
 
